packages/lesswrong/components/editor/EditorFormComponent.tsx

```diff
@@ -239,6 +247,16 @@ class EditorFormComponent extends Component<EditorFormComponentProps,EditorFormC
       this.setState({loading: false})
     }
   }
+  
+  setEditorValue(newValue: string) {
+    const html = markdownToHtmlSimple(newValue)
+    this.setState({
+      markdownValue: newValue,
+      htmlValue: html,
+      ckEditorValue: html
+    })
+    this.state.ckEditorReference?.setData(html)
+  }
```

Comment:
This was the simplest way I could find to allow my form to update the value of a CKEditor field - will probably have to be revisited once we merge in collab editing